admin/services/pupil-census.service.js

```diff
@@ -39,7 +40,10 @@ pupilCensusService.upload = async (uploadFile) => {
       })
   })
   const blobResult = await pupilCensusService.uploadToBlobStorage(csvData)
```

Comment:
I am tempted to remove that as the blobstorage for the current implementation feels very unneeded.
I'll wait for @GuyHarwood comment on that one.